examples/vue-code-file/eslint.config.js

```diff
@@ -1,18 +1,31 @@
-import vueParser from 'vue-eslint-parser';
+import { mergeProcessors } from 'eslint-merge-processors';
+import pluginVue from 'eslint-plugin-vue';
+import processorVueBlocks from 'eslint-processor-vue-blocks';
 import js from '@eslint/js';
 import graphqlPlugin from '@graphql-eslint/eslint-plugin';
 
 export default [
   {
-    files: ['**/*.js', '**/*.vue'],
+    files: ['**/*.js'],
     processor: graphqlPlugin.processor,
     rules: js.configs.recommended.rules,
   },
+  ...pluginVue.configs['flat/recommended'],
   {
     files: ['**/*.vue'],
-    languageOptions: {
-      parser: vueParser,
-    },
+    // `eslint-plugin-vue` will set a default processor for `.vue` files
+    // we use `eslint-merge-processors` to extend it
+    processor: mergeProcessors([
+      pluginVue.processors['.vue'],
+      processorVueBlocks({
+        blocks: {
+          styles: false,
+          customBlocks: true,
```

Comment:
should I remove processing custom blocks? @bmulholland

Reply:
I didn't even know you could do that! They sound cool... and like something that should be out of scope for now.

They're custom, so you'd want to pretty careful with the scenario. Best for someone to have the need and share what it is in more detail. That would probably be a very long tail thing though... honestly probably can always ignore.